src/data/map/metrics.js

```diff
@@ -0,0 +1,1090 @@
+  },
+  {
+    id: 'dem_popbl',
+    title: 'UI_MAP_METRIC_DEM_POPBL',
+    abbrev: '',
+    range: [925, 31423],
+    high_is_good: 1,
+    colors: DEM_COLORS,
+    tab: 'dem',
+    tab_level: null,
+    tooltip: 0,
+    decimals: 2,
+  },
+  {
+    id: 'dem_popas',
+    title: 'UI_MAP_METRIC_DEM_POPAS',
+    abbrev: '',
+    range: [0, 6489],
+    high_is_good: 1,
+    colors: DEM_COLORS,
+    tab: 'dem',
+    tab_level: null,
+    tooltip: 0,
+    decimals: 2,
+  },
+  {
+    id: 'dem_pophi',
+    title: 'UI_MAP_METRIC_DEM_POPHI',
+    abbrev: '',
+    range: [2112, 73920],
+    high_is_good: 1,
+    colors: DEM_COLORS,
+    tab: 'dem',
+    tab_level: null,
+    tooltip: 0,
+    decimals: 2,
+  },
+  {
+    id: 'dem_popm',
+    title: 'UI_MAP_METRIC_DEM_POPM',
+    abbrev: '',
+    range: [3313, 54739],
+    high_is_good: 1,
+    colors: DEM_COLORS,
+    tab: 'dem',
+    tab_level: null,
+    tooltip: 0,
+    decimals: 2,
+  },
+  {
+    id: 'dem_popf',
+    title: 'UI_MAP_METRIC_DEM_POPF',
+    abbrev: '',
+    range: [3218, 50713],
+    high_is_good: 1,
+    colors: DEM_COLORS,
+    tab: 'dem',
+    tab_level: null,
+    tooltip: 0,
+    decimals: 2,
+  },
+  {
+    id: 'dem_thh',
+    title: 'UI_MAP_METRIC_DEM_THH',
+    abbrev: '',
+    range: [1794, 57490],
+    high_is_good: 1,
+    colors: DEM_COLORS,
+    tab: 'dem',
+    tab_level: null,
+    tooltip: 0,
+    decimals: 2,
+  },
+  {
+    id: 'dem_popse',
+    title: 'UI_MAP_METRIC_DEM_POPSE',
+    abbrev: '',
+    range: [598, 10218],
+    high_is_good: 1,
+    colors: DEM_COLORS,
+    tab: 'dem',
+    tab_level: null,
+    tooltip: 0,
+    decimals: 2,
+  },
+  {
+    id: 'dem_popch',
+    title: 'UI_MAP_METRIC_DEM_POPCH',
+    abbrev: '',
+    range: [2007, 28225],
+    high_is_good: 1,
+    colors: DEM_COLORS,
+    tab: 'dem',
+    tab_level: null,
+    tooltip: 0,
+    decimals: 2,
+  },
+]
+
+export const CPAL_FEEDERS = [
+  {
+    id: 1,
+    title: 'Adams',
+  },
+  {
+    id: 2,
+    title: 'Adamson',
+  },
+  {
+    id: 23,
+    title: 'Carter',
+  },
+  {
+    id: 28,
+    title: 'Conrad',
+  },
+  {
+    id: 6,
+    title: 'Hillcrest',
+  },
+  {
+    id: 7,
+    title: 'Jefferson',
+  },
+  {
+    id: 8,
+    title: 'Kimball',
+  },
+  {
+    id: 32,
+    title: 'Madison',
+  },
+  {
+    id: 5,
+    title: 'Molina',
+  },
+  {
+    id: 24,
+    title: 'North Dallas',
+  },
+  {
+    id: 12,
+    title: 'Pinkston',
+  },
+  {
+    id: 13,
+    title: 'Roosevelt',
+  },
+  {
+    id: 14,
+    title: 'Samuell',
+  },
+  {
+    id: 15,
+    title: 'Seagoville',
+  },
+  {
+    id: 25,
+    title: 'Skyline',
+  },
+  {
+    id: 16,
+    title: 'SOC',
+  },
+  {
+    id: 17,
+    title: 'Spruce',
+  },
+  {
+    id: 18,
+    title: 'Sunset',
+  },
+  {
+    id: 21,
+    title: 'White',
+  },
+  {
+    id: 380,
+    title: 'Wilmer-Hutchins',
+  },
+  {
+    id: 22,
+    title: 'Wilson',
+  },
+]
+
+/**
+ * Data metrics in the SEDA data set
+ */
+// export const METRICS = [
```

Comment:
@amygroshek delete?

Reply:
Whatever. Please just merge. We're all on a deadline and I can do cleanup later.